understanding/22/accessible-authentication-enhanced.html

```diff
@@ -45,6 +45,16 @@ <h2>Cognitive function test definition</h2>
         </blockquote>
     </section>
    
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Objective</dt><dd>Make logins possible with less mental effort</dd>
+            <dt>Author task</dt><dd>Don’t make people recognize objects or user-supplied information in order to login</dd>
+            <dt>Key beneficiaries</dt><dd>Users with some cognitive disabilities</dd>
```

Comment:
should probably also include the author task from AA?

```html
<dt>Author task</dt><dd>Don’t make people memorize or transcribe something in order to log in</dd>
```

Reply:
"Don’t make people recognize objects", isn't this going to cause a vast majority of web sites to be in violation? It's quite common to use the "select the pictures with XXX" for user verification. This component includes an alternate way for the user to be verified.

Reply:
Especially for brevity, I've thought of these as listing the differences, so am trying_to list the new ask at the AAA level. The assumption is they've done already met the AA.

Reply:
i would recommend making these "standalone" to be honest. never good to assume that devs are aware of this, particularly if they just landed on the page in isolation (after a google search, for instance, or just because they went directly to it from the main spec)

Reply:
alternatively, explicitly add a sentence that says something like "all the AA ones, and then these..."

Reply:
We discussed this in the meeting. We agreed to keep them short and focused for now, but we'll revisit that after doing the WCAG 2.1 items. There were some concerns about how long they get if you have to include all the A/AA requirements at AAA.